packages/dd-trace/src/appsec/channels.js

```diff
@@ -7,5 +7,6 @@ module.exports = {
   incomingHttpRequestStart: dc.channel('dd-trace:incomingHttpRequestStart'),
   incomingHttpRequestEnd: dc.channel('dd-trace:incomingHttpRequestEnd'),
   bodyParser: dc.channel('datadog:body-parser:read:finish'),
-  queryParser: dc.channel('datadog:query:read:finish')
+  queryParser: dc.channel('datadog:query:read:finish'),
```

Comment:
because of garbage collection issues, if you do `dc.channel().subscribe()` then `dc.channel().publish()` there is a  chance the GC will sweep the channel instance between to the two calls, breaking everything.
Maybe this is only historical and this GC issue doesn't happen anymore, but back when we made this, it was needed.

Reply:
Whatever if this is needed or not, I don't think this is the PR to discuss it, another PR/discussion would be better